pkg/commands/loaders/stash.go

```diff
@@ -65,11 +66,9 @@ outer:
 
 func (self *StashLoader) getUnfilteredStashEntries() []*models.StashEntry {
 	rawString, _ := self.cmd.New("git stash list --pretty='%gs'").DontLog().RunWithOutput()
-	stashEntries := []*models.StashEntry{}
-	for i, line := range utils.SplitLines(rawString) {
-		stashEntries = append(stashEntries, self.stashEntryFromLine(line, i))
-	}
-	return stashEntries
+	return slices.MapWithIndex(utils.SplitLines(rawString), func(line string, index int) *models.StashEntry {
```

Comment:
The lo package unconditionally has an index arg in the callback, but I find I rarely need an index. I'm undecided as to whether I should take the same approach.